optional-requirements.txt

```diff
@@ -1,7 +1,29 @@
-pymysql==1.0.0
-pyodbc==4.0.32
-snowflake-sqlalchemy==1.3.4
+# This file exists for caching within the Docker build
+
+# AWS
+boto3~=1.24.46
```

Comment:
With some new functionality being built around email download of privacy requests I don't think boto3 will be optional.

Reply:
that raises the question of if emails themselves should also be part of the default package though? Are emails a core part of Ops? I suspect we'll need to keep having the discussions on "what is or isn't optional"